.github/workflows/traceability.yml

```diff
@@ -0,0 +1,23 @@
+name: traceability
+
+on:
+  pull_request:
+    branches:
+      - dev
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+  validate-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: AzuObs/github-action-traceability@v1
```

Comment:
This should be updated to your new repo under Neo4j instead of you personal, I assume